pkg/ct/consistency.go

```diff
@@ -22,34 +22,53 @@ import (
 	ct "github.com/google/certificate-transparency-go"
 	ctclient "github.com/google/certificate-transparency-go/client"
 	"github.com/sigstore/rekor-monitor/pkg/util/file"
-	"github.com/sigstore/sigstore/pkg/cryptoutils"
+	"github.com/sigstore/sigstore-go/pkg/root"
+	"github.com/sigstore/sigstore-go/pkg/tuf"
 	"github.com/transparency-dev/merkle/proof"
 	"github.com/transparency-dev/merkle/rfc6962"
 )
 
 const (
-	ctfe2022PubKey = `-----BEGIN PUBLIC KEY-----
-MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEiPSlFi0CmFTfEjCUqF9HuCEcYXNK
-AaYalIJmBZ8yyezPjTqhxrKBpMnaocVtLJBI1eM3uXnQzQGAJdJ4gs9Fyw==
------END PUBLIC KEY-----`
+	ctfe2022KeyID = "dd3d306ac6c7113263191e1c99673702a24a5eb8de3cadff878a72802f29ee8e"
```

Comment:
As you found, there's no good way to do this, as the checkpoint format for CT logs (e.g. https://ctfe.sigstore.dev/2022/ct/v1/get-sth) is quite different. With [static ct](https://github.com/C2SP/C2SP/blob/main/static-ct-api.md), this will change. This is good for now!